rest-service/manager_rest/rest/responses_v2.py

```diff
@@ -32,7 +32,11 @@ def __init__(self, **kwargs):
         self.items = kwargs.get('items')
 
     def __len__(self):
+        if self.items is None:
```

Comment:
Couldn't this be handled better by doing (in the init): kwargs.get('items', [])?